site/en/guides/build/index.md

```diff
@@ -48,32 +15,23 @@ The most common workflow is:
             $ docker pull openthread/environment:latest
             $ docker run -it --rm openthread/environment bash
 
-1.  Within your chosen environment, clone the OpenThread Git repository:
+1.  Within your chosen environment, clone the platform-specific OpenThread Git repository. Taking CC2538 as an example:
 
-           $ git clone https://github.com/openthread/openthread
+           $ git clone https://github.com/openthread/ot-cc2538.git --recursive
 
 1.  From the cloned repository's root directory:
-    1.  Install the GNU toolchain and other dependencies (optional):
+    1.  Install the toolchain:
 
             $ ./script/bootstrap
 
-    1.  Set up the environment:
-
-            $ ./bootstrap
+    1.  Build the configuration:
 
-    1.  Configure and build, using pre-defined platform examples with optional customization via common switches:
-        1.  Modify OpenThread compile-time constants in the selected platform's `/examples/platforms/{platform}/openthread-core-{platform}-config.h` file
-        1.  Build the configuration:
-
-                $ make -f examples/Makefile-{platform} <switches>
+            $ ./script/build <platform-specific-args> <cmake-options>
```

Comment:
Should we use a real working command for a specific platform?

Reply:
Here seems a general guide, there is build example below